src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue_using_dtc.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_moved_to_error_queue_using_dtc : NServiceBusAcceptanceTest
```

Comment:
note: we discussed and spiked potential component tests for the recoverability feature which would allow to test recoverability in better isolation from other pipeline features. This will be handled in a different PR.

For this PR we're ok with testing side effects of non-recoverability features to verify the correct behavior of the recoverability.